app/test_main.py

```diff
@@ -1,3 +1,54 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestAnimalsYearsTransfer:
+    @pytest.mark.parametrize(
+        "cat_years,dog_years,expected_results",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="should return [0, 0] if animals ages are 0's too."
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
```

Comment:
The expected result `[0, 0]` for `cat_years` and `dog_years` both being 14 may not be correct if the logic for ages less than 15 should return a different value. Verify the logic in `get_human_age` to ensure this test case is valid.